compose/django.md

```diff
@@ -25,7 +25,7 @@ and a `docker-compose.yml` file. (You can use either a `.yml` or `.yaml` extensi
 
 3. Add the following content to the `Dockerfile`.
 
-       FROM python:3
+       FROM python:3.8-slim-buster
```

Comment:
Not necessary to be that specific. As a sample, I think we should keep it as simple as possible